Sandbox/src/SandboxApp.cpp

```diff
@@ -218,7 +215,7 @@ class Sandbox : public Hazel::Application
 	}
 };
 
-Hazel::Application* Hazel::CreateApplication()
+Hazel::Scope<Hazel::Application> Hazel::CreateApplication()
 {
-	return new Sandbox();
+	return Hazel::CreateScope<Sandbox>();
```

Comment:
Same thing.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/165/commits/47e7ebf8f1fe6d56aaa93456fc099b014e94f821